File: yh_turtle/scripts/turtle_keyboard.py
#!/usr/bin/python
# -*- coding : utf-8 -*-

import rospy
from geometry_msgs.msg import Twist

pub=rospy.Publisher("turtle1/cmd_vel", Twist,queue_size=20)

def msgCallback(msg):
    pub.publish(msg)

if __name__=="__main__":
    rospy.init_node("turtle_keyboard")
    rospy.Subscriber("cmd_vel", Twist,msgCallback,queue_size=10)

    rospy.spin()

File: yh_turtle/src/turtle_keyboard.cpp
#include "ros/ros.h"
#include "geometry_msgs/Twist.h"

//cmd_vel 토픽 subscirbe==> turtle1/cmd_vel 토픽 publish

ros::Publisher pub;

void msgCallback(const geometry_msgs::Twist::ConstPtr& msg)
{
    pub.publish(*msg);
}

int main(int argc, char** argv)
{
    ros::init(argc, argv, "turtle_keyboard");
    ros::NodeHandle nh;

    pub = nh.advertise<geometry_msgs::Twist>("turtle1/cmd_vel", 10);
    ros::Subscriber sub= nh.subscribe("cmd_vel", 10, msgCallback);

    ros::spin();

    return 0;
}